app/helpers/template_helpers.py

```diff
@@ -57,12 +59,21 @@ def render_template(template, **kwargs):
         get_locale().language, theme or "census"
     )
     page_header_context.update({"title": cookie_session.get("survey_title")})
-    google_tag_mananger_context = get_google_tag_mananger_context()
+    google_tag_manager_context = get_google_tag_manager_context()
     cdn_url = f'{current_app.config["CDN_URL"]}{current_app.config["CDN_ASSETS_PATH"]}'
     contact_us_url = get_contact_us_url(theme, get_locale().language)
+    include_csrf_token = (
+        "POST" in request.url_rule.methods if request.url_rule else False
+    )
+    account_service_url = (
+        f"{CENSUS_BASE_URL}en/start"
+        if not cookie_session or cookie_session.get("account_service_url")
+        else cookie_session.get("account_service_url")
+    )
```

Comment:
Even if there is no cookie sent in a request, we can still check for values in `cookie_session` as we do for `theme` above. So this can be simplified to:
```suggestion
    account_service_url = cookie_session.get("account_service_url", f"{CENSUS_BASE_URL}en/start")
```